ts/Dashboards/Components/KPIComponent/KPIComponent.ts

```diff
@@ -405,6 +406,118 @@ class KPIComponent extends Component {
         this.chart?.destroy();
         super.destroy();
     }
+
+    /**
+     * Calculates value sum based on the provided values.
+     *
+     * @param values
+     * All particular column values.
+     *
+     * @returns
+     * The calculated sum value.
+     *
+     * @internal
+     */
+    private calculateSumValue(values: number[]): number {
+        let calculatedValue = 0;
+
+        for (let i = 0; i < values.length; i++) {
+            const value = values[i];
+
+            if (isNumber(value)) {
```

Comment:
According to TS in the function (and `calculateAverageValue`) the number is required, so do we need to checking it here?

You also check it by `(parsedColumnValues?.every(isNumber)) {`